www/utils/api.ts

```diff
@@ -186,4 +190,66 @@ export class API {
       return null;
     }
   }
+
+  async addReaction(
+    conversationId: string,
+    messageId: string,
+    reaction: Exclude<Reaction, null>,
+  ): Promise<{ status: string }> {
+    try {
+      const response = await fetch(
+        `${this.url}/api/reaction/${messageId}?user_id=${this.userId}&conversation_id=${conversationId}&reaction=${reaction}`,
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        },
+      );
+      if (!response.ok) {
+        throw new Error("Failed to add reaction");
+      }
+
+      return await response.json();
+    } catch (error) {
+      console.error("Error adding reaction:", error);
+      throw error;
+    }
+  }
+
+  async getReaction(
```

Comment:
When would this need to be used if we can get the Reaction data by just getting the messages?

Reply:
Good point, removing this method.